### DISCUSSION TITLE
Edit config before deploy on QNAP container station

### DISCUSSION THREAD

**Discussion by domfitty** [TwiN/gatus#503] [Q&A] [answered]
I'm trying gatus on a QNAP NAS using container station but I'm unable to edit config.yaml so the application starts only with default endpoints.
Any ideas?

**Comment by TwiN** [comment, marked as answer]
You probably didn't mount your config.yaml volume/file properly.

I can't speak for your specific container station, but here's what it looks like on my NAS: 
![image](https://github.com/TwiN/gatus/assets/15699766/5f4949cd-24e1-4138-ad3c-4d4e8a3fb2db)

**Reply by domfitty** [reply]
Thanks, I will try
